reports/README.md

```diff
@@ -0,0 +1,55 @@
+# Explanation
+
+The code in this folder (`reports`) is used to generate CSV files for getting daily/weekly/monthly active users for all Ignite team (ipfs-gui) projects.
+
+The CSV content is then loaded into it's respective sheet at https://docs.google.com/spreadsheets/d/1xq36kjThObEaRKzb3VRtXEs9RgM-bfgfjGbi1vbPUiE/edit#gid=755468744
+
+The charts in the "Charts" sheet are loaded in our Notion page at https://www.notion.so/pl-strflt/Ignite-IPFS-GUI-Tools-3bc1c1bf54d74f928bf11ef59c876b74#b6970aa92e914114848fbddd84eab2ba
+
+NOTE: The below instructions do not need to be followed once our GitHub Action is merged. The data will update according to the scheduled GitHub Action at [`../.github/workflows/metrics-update.yml`](../.github/workflows/metrics-update.yml)
+
+## With google sheets authentication
+
+You need the following ENV vars set properly:
+
+* `GOOGLE_CREDENTIALS` - Your 'JSON service account key' file stringified into a single line
```

Comment:
The service account i created under SgtPooki i would hope. Not too concerned about edit history since it should be limited to the tool once this is merged

Reply:
it shows the service account. below shows an edit by me directly, and by the google service account 

![image](https://user-images.githubusercontent.com/1173416/215838366-9b892e35-09c5-4f5a-aeba-78d36d2a1940.png)